canonical-web-and-design/snapcraft.io
# Builds table

Fixes #2355

Adds basic view for builds page using maas react table component.

https://snapcraft-io-canonical-web-and-design-pr-2359.run.demo.haus/speed-test/builds


### QA
- ./run or demo
- go to new Builds page of any snap built on BSI
- see the list of the builds with some basic info

<img width="1119" alt="Screenshot 2019-11-18 at 15 18 12" src="https://user-images.githubusercontent.com/83575/69059878-b0345780-0a16-11ea-9316-1c2d1959931b.png">
